client/libs/react-app-context.jsx

```diff
@@ -3,12 +3,16 @@ import {React} from 'meteor/react-runtime';
 
 export function applyContext(context, _actions) {
   const actions = {};
-  for (var key in _actions) {
-    const actionMap = _actions[key];
-    for (var actionName in actionMap) {
-      actionMap[actionName] = actionMap[actionName].bind(null, context);
+  for (let key in _actions) {
```

Comment:
I think for ... of for object is not available in Meteor yet. I hope it's in stage-2 (in babel 6 as well).
Anyway, I am moving this code into a NPM module.